src/app/easy-spinner/components/spinner.component.spec.ts

```diff
@@ -1,14 +1,13 @@
 import { TestBed, async } from '@angular/core/testing';
 import { By } from '@angular/platform-browser'
-import { Observable } from 'rxjs/Observable';
-import 'rxjs/add/observable/of';
+import { Observable, of } from 'rxjs';
 import { SpinnerComponent } from './spinner.component';
 
 import { SpinnerService } from '../services/spinner.service';
 
 
 class MockService {
-    spinnerControl = Observable.of(false);
+    spinnerControl = of(false);
```

Comment:
Do we still need Observable.of, or does the operator work on it own in v6?

Reply:
Nope.  There's no Observable class in v6
https://github.com/ReactiveX/rxjs/blob/master/MIGRATION.md